Platform/ApollolakeBoardPkg/BoardConfig.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, *args, **kwargs):
 
 		# BIT0:Serial  BIT1:USB KB
 		# Support serial port input console by default
-		self.CONSOLE_IN_DEVICE_MASK  = 0x00000001
+		self.CONSOLE_IN_DEVICE_MASK  = 0x00000003
```

Comment:
I think it will give good experience for the user to use USB keyboard, instead that they tried and found it doesn't work, then enable it later.

Reply:
I vote to keep SBL small and fast by default.  
USB keyboard is for debug purpose only.  And USB enumeration takes lots of time.   I prefer keeping current behavior,  disable by default.

Reply:
Let us decouple this two changes.    For the 1st fix,  it is great and we should merge it in.
For the 2nd change to enable USB by default, since it is debatable,  let us keep it separate.

Reply:
Sure. just roll back this change.